url.bs

```diff
@@ -246,9 +305,28 @@ a <var>percentEncodeSet</var>, run these steps:
    <td>"<code>‽%25%2E</code>"
    <td>0xE2 0x80 0xBD 0x25 0x2E
   <tr>
-   <td><a for="code point">UTF-8 percent-encode</a> <var>input</var> using the
+   <td rowspan=3><a for="code point">Percent-encode after encoding</a> with <a>Shift_JIS</a>,
+   <var>input</var>, and the <a>userinfo percent-encode set</a>
+   <td>U+0020
+   <td>"<code>%20</code>"
+  <tr>
+   <td>U+2261 (≡)
+   <td>"<code>%81%DF</code>"
+  <tr>
+   <td>U+203D (‽)
+   <td>"<code>%26%238253%3B</code>"
+  <tr>
+   <td><a for=string>Percent-encode after encoding</a> with <a>Shift_JIS</a>, <var>input</var>, the
+   <a>userinfo percent-encode set</a>, and true
+   <td>"<code>1+1 ≡ 2%20‽</code>"
+   <td>"<code>1+1+%81%DF+2%20%26%238253%3B</code>"
+  <tr>
+   <td rowspan=2><a for="code point">UTF-8 percent-encode</a> <var>input</var> using the
```

Comment:
Nit: Perhaps move the UTF-8 examples up above Shift-JIS since they are the far more common example.